### PR TITLE
Set properties in initializer

### PR DESCRIPTION
**Pull request by jamonholmgren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
-# This repository is no longer maintained.
-
-If you'd like to take over Parsistence, get in touch with us. But we are no longer using it and have no plans to continue supporting it.
-
-- The ClearSight Studio team
-
```
Comment (jamonholmgren) [inline README.md]: It's *kinda* maintained now...again. :)

**Comment by jamonholmgren** [comment]
Since we don't have tests, can @silasjmatson use this branch in his current app to see if it works?

**Comment by jamonholmgren** [comment]
@silasjmatson bump.

**Comment by silasjmatson** [comment]
@jamonholmgren It works.

**Comment by jamonholmgren** [comment]
Sweet!

**Pull request merged**